webapp/sass/layout/_content.scss

```diff
@@ -23,9 +23,9 @@
         .search-btns {
             display: none;
         }
-        .header-list__members {
+        .header-list__members,
+        .header-list__pinned-posts {
             margin-right: -18px;
-            float: right;
```

Comment:
I had to remove the float right as the icons wouldn't stay on the same line when resizing down the screen. I played around to see if it had any undesired effects but didn't find anything.